### PR TITLE
GH-6: Dead Letter Queue Processing

### LINKED ISSUE #6: Document how to re-process messages from DLQ
As a developer, I'd like to document how to re-process messages from DLQ with sample.

### PR DESCRIPTION
**Pull request by garyrussell**
Resolves #6

### CODE REVIEW

File: spring-cloud-stream-binder-rabbit-docs/src/main/asciidoc/contributing.adoc [hunk 1]
```diff
@@ -1,4 +1,4 @@
-[[contributing]
+[[rabbit-binder-contributing]]
```
Comment (mbogoevici) [inline spring-cloud-stream-binder-rabbit-docs/src/main/asciidoc/contributing.adoc]: Do we need to rename this? It does not get aggregated anywhere.

File: spring-cloud-stream-binder-rabbit-docs/src/main/asciidoc/overview.adoc [hunk 2]
```diff
@@ -38,13 +38,25 @@ For each consumer group, a `Queue` will be bound to that `TopicExchange`.
 Each consumer instance have a corresponding RabbitMQ `Consumer` instance for its group's `Queue`.
 For partitioned producers/consumers the queues are suffixed with the partition index and use the partition index as routing key.
 
+Using the `autBindDlq` option, you can optionally configure the binder to create and configure dead-letter queues (DLQs) (and a dead-letter exchange `DLX`).
```
Comment (mbogoevici) [inline spring-cloud-stream-binder-rabbit-docs/src/main/asciidoc/overview.adoc]: `autoBindDlq`

File: spring-cloud-stream-binder-rabbit-docs/src/main/asciidoc/contributing.adoc (hunk 1 above)
Comment (garyrussell) [inline spring-cloud-stream-binder-rabbit-docs/src/main/asciidoc/contributing.adoc]: Just for safety 😄

**Comment by garyrussell** [comment]
Need to add coverage for partitioned destinations.

File: spring-cloud-stream-binder-rabbit-docs/src/main/asciidoc/contributing.adoc (hunk 1 above)
Comment (mbogoevici) [inline spring-cloud-stream-binder-rabbit-docs/src/main/asciidoc/contributing.adoc]: cool :)

**Pull request closed without merging**